### ISSUE TITLE
Still the same

### ISSUE THREAD

**Issue by sm6usu** [ExpertSDR3/ExpertSDR3-BUG-TRACKER#482]
Hallo again Remote in New Beta was no different to earlier problem with about 50 % power level Had a mail this was fixed.?
73 sm6usu Svante

**Comment by VK6NX** [comment]
And this is why you have decided to open new case. With very informative Topic name and description.

Let me join you.

"Still the same". Yep. I feel better now.

**Comment by ExpertSDR3** [comment]
This problem is solved by updating Starter

**Comment by sm6usu** [comment]
Thank you will try today
All the best sm6usu Svante

Den tis 1 nov. 2022 kl 07:09 skrev EE ***@***.***>:

**Comment by sm6usu** [comment]
Ok sorry I had this issue discuss earlier 2 times. But any how I try to explain again. I try to use remote from my Laptop to my stationary pc. Laptop windows 10 navidia rtx 3070. Stationary pc Windows 11. Both running without problem with my sdr2 dx in version 3 and now the new Beta version. I can also running the laptop as a remote station to my windows 11 pc. But of some reason ,power level is only from about 43 to about 57 % when I tuning even with tuning slider at maximum. I have tried to make qso and it is impossible they simply do not hear me. I did get a mail from ee team that they found the problem and that was in a server and was now fixed. Therefore closed my issue.  Ok nice.  I did a new try (before beta) and no change on my side. I reopened the case and tell them and also said that maybe I have to wait for a newer release. They come back to me and cofirmed that I have to have the release which comes after the 24 Oktober. No problem. Now I download the Beta version and no change at all about the powerlevel. I have no hurry. I just want them to know. I understand they work hard to solve all problem we find in Alfa and now in Beta. I am very happy with Software and Hardware. Maybe I was a little tired when I wrote about the issue the last time and it end up a little short. Sorry about that. Now I have got a new mail from EE that I have to update to a new starter. So I will try that in a couple of hours and hope for the best. 
All the best 73 de sm6usu Svante
